demisto_sdk/commands/upload/tests/uploader_test.py

```diff
@@ -879,6 +959,7 @@ def test_upload_with_skip_verify(self, mocker, input):
         # prepare
```

Comment:
Please add test that validate that skip_verify wasn't sent in case of server version < 6.5.0

Reply:
done